sqlbrite-kotlin/src/main/java/com/squareup/sqlbrite3/extensions.kt

```diff
@@ -64,6 +66,7 @@ inline fun <T> Observable<Query>.mapToOneOrDefault(default: T, noinline mapper:
  *
  * @param mapper Maps the current [Cursor] row to `T`. May not return null.
  */
+@RequiresApi(24)
```

Comment:
Should we use the constant instead?

Reply:
I tend to prefer the number since it's what used on API docs and in the lint warnings. It also avoids having to do the mental mapping from name/letter to number (or just hitting CMD+B) after a long day.